conans/client/conan_command_output.py

```diff
@@ -125,15 +125,14 @@ def _grab_info_data(self, deps_graph, grab_paths):
 
             # Paths
             if isinstance(ref, ConanFileReference) and grab_paths:
-                package_layout = self._cache.package_layout(ref, conanfile.short_paths)
-                item_data["export_folder"] = package_layout.export()
-                item_data["source_folder"] = package_layout.source()
+                ref_layout = self._cache.get_ref_layout(ref)
```

Comment:
@czoido @memsharded should I have used `get_latest_` instead?

Reply:
Same as above, ref already has a revision, get_ref_layout will check and assert.